android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationListenerService.java

```diff
@@ -93,6 +98,14 @@ private void sendNotification(ReactApplicationContext context, Bundle bundle) {
         if (bundle.getString("contentAvailable", "false").equalsIgnoreCase("true")) {
             jsDelivery.notifyRemoteFetch(bundle);
         }
+
+        Log.e(LOG_TAG, "RNPushNotificationListenerService.sendNotification: " + bundle);
+
+        if (!isRunning) {
```

Comment:
@mikelambert I was simply restoring the old behaviour that was removed, as it cause a huge regression in my app. Our android app stopped showing notifications.

If you ask me we should investigate exactly how `PushNotificationIOS` behaves and implement the same.

I think this is it, but I'll take another look just to make sure and let you know.